src/main/java/com/rylinaux/plugman/util/BukkitCommandWrap.java

```diff
@@ -177,92 +125,193 @@ public void sync() {
 
     public void unwrap(String command) {
         if (this.nmsVersion == null) return;
-        if (this.minecraftServerClass == null) try {
-            this.minecraftServerClass = Class.forName("net.minecraft.server." + this.nmsVersion + ".MinecraftServer");
-        } catch (ClassNotFoundException e) {
-            try {
-                this.minecraftServerClass = Class.forName("net.minecraft.server.MinecraftServer");
-            } catch (ClassNotFoundException classNotFoundException) {
-                classNotFoundException.printStackTrace();
-                classNotFoundException.addSuppressed(e);
-                return;
-            }
-        }
-        if (this.getServerMethod == null) try {
-            this.getServerMethod = this.minecraftServerClass.getMethod("getServer");
-            this.getServerMethod.setAccessible(true);
-        } catch (NoSuchMethodException e) {
-            e.printStackTrace();
-            return;
-        }
 
-        Object server;
+        if (!resolveMcServerClass()) return;
+
+        if (!resolveGetServerMethod()) return;
+        Object server = getServerInstance();
+
+        if (!resolveVanillaCommandDispatcherField()) return;
+        Object commandDispatcher = getCommandDispatcher(server);
+
+        if (!resolveBField()) return;
+
+        CommandDispatcher b = getDispatcher(commandDispatcher);
+        if (b == null) return;
```

Comment:
I'd like to switch to meaningful names.

Would you be willing to do it for me?

Reply:
I actually don't know what they are supposed to mean. If you know, I can rename them no problem.

Reply:
Well... In this case, you can rename `b` to commandDispatcher xd

But it's fine. I'll do it at some point